src/io/platform/f3xx/f302x8/CANf302x8.cpp

```diff
@@ -168,6 +180,10 @@ CANMessage* CANf302x8::receive(CANMessage* message, bool blocking) {
     }
 }
 
+void CANf302x8::setCANFilterId(uint32_t identifier) {
+    this->identifier = identifier;
```

Comment:
This identifier would not be explicitly needed with the proposed method for adding filters.